Dockerfile

```diff
@@ -0,0 +1,99 @@
+FROM debian:12.9-slim
+
+WORKDIR /build
+
+# Install utils
+RUN apt update \
+    && apt install wget gpg unzip git -y \
+    && dpkg --add-architecture i386
+
+# Install wine
+RUN mkdir -pm755 /etc/apt/keyrings \
+    && (wget -O - https://dl.winehq.org/wine-builds/winehq.key | gpg --dearmor -o /etc/apt/keyrings/winehq-archive.key -) \
+    && wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/debian/dists/bookworm/winehq-bookworm.sources \
+    && apt update \
+    && apt install --no-install-recommends winehq-stable -y
+
+# Clean up APT
+RUN apt clean \
+    && rm -rf /var/lib/apt/lists/* \
+
+WORKDIR /build/tools/
+
+# Install cmake windows
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.31.6/cmake-3.31.6-windows-x86_64.zip \
+    && unzip cmake-3.31.6-windows-x86_64.zip -d /build/tools/ \
+    && mv /build/tools/cmake-3.31.6-windows-x86_64 /build/tools/cmake
+
+# Install git windows
+RUN wget https://github.com/git-for-windows/git/releases/download/v2.49.0-rc1.windows.1/MinGit-2.49.0-rc1-64-bit.zip \
+    && unzip MinGit-2.49.0-rc1-64-bit.zip -d /build/tools/ \
+    && mv /build/tools/cmd /build/tools/git
```

Comment:
It seems like this dependency exists solely because of the way the cmake files in `/cmake/` are using `FetchContent_Declare`.

It's out of the scope of this PR/change, but would we be better off just using git submodules, for these git-based dependencies?

Reply:
CMake for Linux/Docker does not have the Generator for NMake included. So the workaround is to install the Windows version of CMake. This version needs git. But the build in Git in Linux would not do it wont work. Git Wubmodules are a viable alternative. But now you need to not forget to pull them as well as this is not standard behavior of git when cloning the repo. I like the one point of glass of cmake to manage dependency's for me.